src/current/_includes/releases/v24.1/v24.1.10.md

```diff
@@ -54,7 +54,7 @@ Release Date: January 9, 2025
 - Using more than one `DECLARE` statement in the definition of a user-defined function (UDF) now correctly declares additional variables. [#135739][#135739]
 - Fixed an issue where corrupted table statistics could cause the `cockroach` process to crash. [#136040][#136040]
 - Fixed a bug that could cause the password for the `root` user to be deleted while upgrading to v24.1. This bug only affected clusters that were initially created with v22.2 or earlier. The same bug could also cause the `defaultdb` and `postgres` databases to be recreated during the upgrade to v24.1 if they had been previously deleted. [#136074][#136074]
-- `CLOSE CURSOR` statements are now allowed in read-only transactions, similar to Postgres. The bug had been present since at least v23.1. [#137790][#137790]
+- `CLOSE CURSOR` statements are now allowed in read-only transactions, similarly to PostgreSQL. The bug had been present since at least v23.1. [#137790][#137790]
```

Comment:
In hindsight, I think `similar` was actually correct! I've adjusted on my own (identical) release note as well. Sorry about that.

Reply:
updated in latest commit!